Integrations/integration-Cymon.yml

```diff
@@ -0,0 +1,543 @@
+description: Analyzes suspicious hashes, URLs, domains and IP addresses
+configuration:
+  - display: Trust any certificate (unsecure)
+    name: unsecure
+    defaultvalue: ""
+    type: 8
+    required: false
+script:
+  script: |
+    ''' IMPORTS '''
+    import datetime
+    import time
+    import requests
+    import json
+    import os
+    import re
+
+    # disable insecure warnings
+    requests.packages.urllib3.disable_warnings()
+
+    if not demisto.params().get('useProxy', False):
+        del os.environ['HTTP_PROXY']
+        del os.environ['HTTPS_PROXY']
+        del os.environ['http_proxy']
+        del os.environ['https_proxy']
+
+    ''' GLOBAL VARS '''
+    SERVER_URL_V1='https://www.cymon.io:443/api/nexus/v1'
+    SERVER_DASHBOARD_URL_V1='https://www.cymon.io:443/api/dashboard/v1'
+    SERVER_URL_V2 = 'https://api.cymon.io/v2/ioc/search'
+
+    USE_SSL = True if demisto.params().get('unsecure') else False
```

Comment:
@MarkShleifer I think the condition should be reversed

Reply:
Done.